draft-ietf-quic-manageability.md

```diff
@@ -181,6 +181,21 @@ used to separate QUIC packets. The length header field is variable length and
 its position in the header is also variable depending on the length of the
 source and destionation connection ID. See Section 4.6 of {{QUIC-TRANSPORT}}.
 
+## Use of Port Numbers
+
+Applications that have a mapping for TCP as well as QUIC are expected to
+use the same port number for both services. However, similar as today
```

Comment:
RFC3556 say "IANA strives to assign only one assigned port number per service or application" which implies that you use the same port for the same service even if a different transport is used. What is wrong about this sentence is that I talk about two/both services because it's the same service effectively